tasks/new_e2e_tests.py

```diff
@@ -189,6 +189,7 @@ def run(
         print(
             f'To run this test locally, use: `{command}`. '
             'You can also add `E2E_DEV_MODE="true"` to run in dev mode which will leave the environment up after the tests.'
+            '\nYou can troubleshoot e2e test failures with this documentation: https://datadoghq.atlassian.net/wiki/spaces/ADX/pages/3492283118/Troubleshoot+an+e2e+test+failing+on+the+CI'
```

Comment:
```suggestion
            '\nYou can troubleshoot e2e test failures with this documentation: https://datadoghq.atlassian.net/wiki/x/7gIo0'
```
I was told once in the past that this link was better because it would remain unchanged if the page title changes. I have no idea if it's still true, and it's maybe quite sure this page won't change its title. In any case I'm sharing this here just in case.